### ISSUE TITLE
A11y_Visual Studio Code_ search side bar options_Info and Relationships:Voice Over is  announcing as "Search Include Patterns" and "Search Exclude Patterns" instead of Files to include and Files to Exclude Respectively. 

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode#131910]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#Infoand Relationships;#MAC;#AINotInScope;#VoiceOver;#Benchmark-VisualStudioCode-MAC-Aug2021

### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Info and Relationships

### Repro Steps:​
1.Open Visual Studio Code editor
2.Tab till "Toggle Search Editor" in the Left pane and Press Enter.
3.Tab till "Search Include Patterns" and "Search Exclude Patterns" .
4. Verify whether the Voice over is announcing the visible names "Search Include Patterns" and "Search Exclude Patterns"   or not.
### Actual Result:
Voice Over is  announcing as "Search Include Patterns" and "Search Exclude Patterns" instead of Files to include and Files to Exclude Respectively. 
### Expected Result: ​
Voice Over should not announce  as  "Files to include" and "Files to Exclude ".

### User Impact:
Screen Reader users will find difficult in finding the Correct label name if the Voice Over is  announcing as "Search Include Patterns" and "Search Exclude Patterns" instead of Files to include and Files to Exclude Respectively.

**Comment by Latha2564** [comment]
![A11y_Visual studioCode_Voice Over is announcing as  Search include and Exclude  patterns](https://user-images.githubusercontent.com/89508906/132457202-349f4073-fc94-44bd-8ef9-0a06de6b0a06.gif)

**Comment by JacksonKearl** [comment]
cc @roblourens made sense to me to have aria label match title, lmk if theres a reason it shouldn't.

**Comment by Latha2564** [comment]
![Untitled](https://user-images.githubusercontent.com/89508906/133738906-257604d5-a789-4e41-a47d-1a7a712b9e88.gif)

**Comment by Latha2564** [comment]
![Uploading A11y_VSCode_131910.gif…]()

**Comment by Latha2564** [comment]
![A11y_VSCode_131910](https://user-images.githubusercontent.com/89508906/133740608-f114a0d9-7853-469a-b1aa-ab3adeff0f32.gif)

**Comment by Latha2564** [comment]
![131910](https://user-images.githubusercontent.com/89508906/133741450-0fbb45f4-1797-4dbe-99a5-7f833b88af51.gif)

**Comment by Latha2564** [comment]
![131910](https://user-images.githubusercontent.com/89508906/133742005-ae508833-ad7d-4639-8bdb-1714f6362faf.gif)

**Comment by roblourens** [comment]
@Latha2564 these gifs that you've added to several issues are extremely compressed and really not helpful. And this issue is fixed.
